dppylib/__init__.py

```diff
@@ -72,7 +77,7 @@ def diff_files(db, collection, file_info):
         logger.info('{FILE} does not exist in the database. Importing.'.format(FILE=file_path))
         import_data(db, collection, file_info)
     else:
-        if db_data['mtime'] != file_info['mtime'] or db_data['size'] != file_info['size']:
```

Comment:
Under the above `probe` criteria, the `mtime` condition was never activated. Even if it was, it would not have been right. Because some of our regenerated files have the same content as that of the old. We want such regenerated files to be skipped.